packages/playground/README.md

```diff
@@ -1,3 +1,37 @@
-# playground
+# Playground
 
-This package is not published, and is just used to verify a standalone app
\ No newline at end of file
+Playground is a sample standalone application that shows usage of React Native for Windows vNext with several test scenarios. 
+
+## Launching the app
+1. Make sure your development machine has been set up with all the system requirements and dependencies mentioned [here](https://github.com/microsoft/react-native-windows/blob/master/vnext/docs/GettingStarted.md). Make sure a browser is launched and running.
```

Comment:
It's not only for playground, but It's better to ask user to set Developer Mode  in GettingStarted.md

Reply:
Done.